cluster/gce/util.sh

```diff
@@ -1272,7 +1272,17 @@ function test-teardown {
     --project "${PROJECT}" \
     --quiet \
     "${NODE_TAG}-${INSTANCE_PREFIX}-nodeports" || true
-  "${KUBE_ROOT}/cluster/kube-down.sh"
+  if [[ ${MULTIZONE:-} == "true" ]]; then
+      local zones=( ${E2E_ZONES} )
+      # tear them down in reverse order, finally tearing down the master too.
+      for ((zone_num=${#zones[@]}-1; zone_num>0; zone_num--))
+      do
+	  KUBE_GCE_ZONE="${zones[zone_num]}" KUBE_USE_EXISTING_MASTER="true" "${KUBE_ROOT}/cluster/kube-down.sh"
+      done
+      KUBE_GCE_ZONE="${KUBE_GCE_ZONE}" "${KUBE_ROOT}/cluster/kube-down.sh"
```

Comment:
it seems that `KUBE_GCE_ZONE` is only set in `hack/jenkins/e2e.sh`, so I'm worried that runs will fail here. (The cluster/gce scripts read `KUBE_GCE_ZONE` but have a default if it's not set.)